frontend/src/scenes/data-management/events/eventDefinitionsTableLogic.ts

```diff
@@ -281,6 +283,50 @@ export const eventDefinitionsTableLogic = kea<
                 )
             )
         },
+        loadEventDefinitionsSuccess: () => {
+            if (cache.eventsStartTime !== undefined) {
+                eventUsageLogic
+                    .findMounted()
+                    ?.actions.reportDataManagementEventDefinitionsPageViewed(
+                        performance.now() - cache.eventsStartTime,
+                        values.eventDefinitions.results.length
+                    )
+                cache.eventsStartTime = undefined
+            }
+        },
+        loadPropertiesForEventSuccess: () => {
+            if (cache.propertiesStartTime !== undefined) {
+                eventUsageLogic
+                    .findMounted()
+                    ?.actions.reportDataManagementEventDefinitionsPageNestedPropertiesViewed(
+                        performance.now() - cache.propertiesStartTime
+                    )
+                cache.propertiesStartTime = undefined
+            }
+        },
+        loadEventDefinitionsFailure: ({ error }) => {
+            if (cache.eventsStartTime !== undefined) {
+                eventUsageLogic
+                    .findMounted()
+                    ?.actions.reportDataManagementEventDefinitionsPageViewed(
```

Comment:
Result length being `-1` here seems weird. Would probably throw off future metrics.

Wondering if this event should be `Data Management Event Definitions Load Failed` and then we'd have a seperate event for the `success` case, and then we would lean on `pageview` to track page views.